### ISSUE TITLE
Glitchium

### ISSUE THREAD

**Issue by Shazuli** [IntercraftMC/IntercraftCore-Fabric#3]
Glitchium is a rare type of "metal" that is found in small batches as a block covering surfaces when the server goes under a certain TPS (no idea if that's a good idea to farm it, just thought it would be a fun way to do it). Or can just generate covering surfaces. Generally, 1-2 tiny dust can drop when broken, more with Fortune. Has a animated texture making its appearance looking "glitched" and with tooltip of common Java error exception, such as `NullPointerException`, `ArrayIndexOutOfBoundsException`, `NaN`, etc.
### Usages
- [ ] Hard Anvil
An Anvil with more durability than a regular Anvil. Allows illegal combinations of enchantments on tools such as Fortune and Silk Touch or Protection and Fire Protection. Will always set the required cost to the maximum of the finished item, making working with the item further impossible.
- [ ] Sky Block
Basically a copy of OpenBlock's [Sky Block](https://ftbwiki.org/Sky_Block). Rendering the current dimension's skybox on it, with extra functionality such as; natural light under it, blocks Beacon Beams from reaching the sky allowing for hidden Beacons not clogging up the sky.
